docs/reference/migration/migrate_8_3.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[migrating-8.3]]
+== Migrating to 8.3
+++++
+<titleabbrev>8.3</titleabbrev>
+++++
+
+This section discusses the changes that you need to be aware of when migrating
+your application to {es} 8.3.
+
+See also <<release-highlights>> and <<es-release-notes>>.
+
+coming::[8.3.0-SNAPSHOT]
+
+
+[discrete]
+[[breaking-changes-8.3]]
+=== Breaking changes
+
+There are no breaking changes in {es} 8.3.
+
+
+[discrete]
+[[deprecated-8.3]]
+=== Deprecations
+
+The following functionality has been deprecated in {es} 8.3
+and will be removed in a future version.
+While this won't have an immediate impact on your applications,
+we strongly encourage you take the described steps to update your code
+after upgrading to 8.3.
+
+To find out if you are using any deprecated functionality,
+enable <<deprecation-logging, deprecation logging>>.
+
+
+[discrete]
+[[deprecations_83_cluster_and_node_setting]]
+==== Cluster and node setting deprecations
+
+[[configuring_bind_dn_in_an_ldap_or_active_directory_ad_realm_without_corresponding_bind_password_deprecated]]
+.-| Configuring a bind DN in an LDAP or Active Directory (AD) realm without a corresponding bind password is deprecated
```

Comment:
The `-|` characters are appearing in the output, so they seem to be unnecessary:
```suggestion
.Configuring a bind DN in an LDAP or Active Directory (AD) realm without a corresponding bind password is deprecated
```

Reply:
I fixed that by editing the original source 85326.yaml to use the correct `|-` instead of `-|`, in https://github.com/elastic/elasticsearch/pull/87294/commits/2aaa3fa1598cd5a565a8a1e6b1d622e4d90e4266